packages/core/src/node/bunyan-logger-server.ts

```diff
@@ -42,11 +42,12 @@ export class BunyanLoggerServer implements ILoggerServer {
         if (['trace', 'debug', 'info', 'warn', 'error', 'fatal'].indexOf(logLevel) < 0) {
             logLevel = 'info';
         }
+    }
 
-        this.loggers.push(bunyan.createLogger({
-            name: 'Theia',
-            level: logLevel
-        }));
+    create(serveroptions: object | undefined) {
```

Comment:
that's also my wonder, why do we need an extra remote call on the startup if we can statically configure it on the backend? but if we go with an extra call at least all methods in `Logger` should wait till `create` is resolved